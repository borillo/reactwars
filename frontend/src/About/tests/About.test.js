const NUMBER_OF_DIRECTORS = 5;

describe("About", () => {
  beforeEach(() => {});

  test("should present wellcome information", () => {});

  test("should show associated directors", () => {});

  test("should match snapshot", () => {});
});
