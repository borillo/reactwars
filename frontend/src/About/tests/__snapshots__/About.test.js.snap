// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `undefined`;
